pandas/io/formats/style.py

```diff
@@ -72,32 +73,46 @@ class Styler(object):
 
     Notes
     -----
-    Most styling will be done by passing style functions into
-    ``Styler.apply`` or ``Styler.applymap``. Style functions should
-    return values with strings containing CSS ``'attr: value'`` that will
-    be applied to the indicated cells.
 
-    If using in the Jupyter notebook, Styler has defined a ``_repr_html_``
```

Comment:
Why did you remove this? I think it's helpful to note that the class defines `_repr_html_`.

Reply:
I didn't remove it - it's now in line 109.